cylc/flow/suite_files.py

```diff
@@ -1305,3 +1339,18 @@ def link_runN(latest_run):
         run_n.symlink_to(latest_run)
     except OSError:
         pass
+
+
+def search_install_source_dirs(flow_name: str) -> Optional[Path]:
+    """Return the path of a workflow source dir if it is present in the
+    'global.cylc[install]source dirs' search path."""
+    search_path: List[str] = glbl_cfg().get(['install', 'source dirs'])
```

Comment:
search_paths might be easier to understand?

Reply:
We've settled on "installing" a "workflow" from a "source dir" to a "run dir".

So "source dir**s**" makes sense.